### ISSUE TITLE
Card Subhead field failing color contrast ratio in Dark Mode

### ISSUE THREAD

**Issue by acoleman** [unlcms/project-herbie#911]
The Subhead field on Card components with "Lighter Gray" or "Lightest Gray" background color fail color contrast ratio in Dark Mode.
